packages/documentation/copy/en/handbook-v2/Classes.md

```diff
@@ -909,6 +909,38 @@ const MyHelperObject = {
 };
 ```
 
+### Type-Checking Static Members of a Class
+
+Under the hood, each class is an object (really a function) with a prototype property.  You can define a type for the class's static members, which will be properties of the class (not instances of it).

```

Comment:
I'm not totally sure that this section is worth including, at least not as-is.

Are you hoping to have an example that shows how to work around the lack of `class Vehicle extends static VehicleTypeStatic`? There's probably a simpler way to do that, if so; I'd think we'd want to be pretty targeted about showing the difference between the the "value" of the class and the "type" of a class instance. Not sure if this doc already includes some of that language, though.